samcli/cli/hidden_imports.py

```diff
@@ -1,3 +1,6 @@
+"""
+Keeps list of hidden/dynamic imports that is being used in SAM CLI, so that pyinstaller can include these packages
+"""
```

Comment:
A quick question: what was the reason for us to move this file into samcli.cli?

Reply:
After discussion this change is required for the following reason: in order to access it for the new _dynamic_import logic, this file needs to be in SAM CLI code